assets/css/amp-default.css

```diff
@@ -3,3 +3,11 @@
 	/** Force the image into a box of fixed dimensions and use object-fit to scale. **/
 	object-fit: contain;
 }
+
+#amp-image-lightbox-1 img {
```

Comment:
I don't think we should be targeting `img` elements in selectors. I should think that only targeting `amp-img` should be done since `img` should really be the domain of the `amp-img` to control?

Reply:
Looks like `amp-image-lightbox` component is empty by default and clicking on image which is targeting this lightbox will insert the `img` element into the `amp-image-lightbox`, the `amp-image-lightbox` component doesn't include an additional `amp-img` component within, targeting `amp-img` doesn't seem to be possible in this case.
For example after clicking on an an image the lightbox HTML looks like this:
![screen shot 2018-05-21 at 10 34 06 pm](https://user-images.githubusercontent.com/3294597/40326414-ce995022-5d47-11e8-9166-b81dc0295098.png)
Thoughts?